qemu/tests/nested_test.py

```diff
@@ -29,9 +29,11 @@ def run(test, params, env):
     setup_bridge_sh = "/root/setup_bridge.sh"
 
     def get_live_vms(env):
+        login_timeout = params.get_numeric("login_timeout", 360)
         live_vms = []
         for vm in env.get_all_vms():
             if vm.is_alive():
+                vm.wait_for_login(timeout=login_timeout)
```

Comment:
I think this change can not fix bz 1847824: 'No route to host' error.
1. Call `wait_for_login`(generate_invent_file) before `wait_for_get_address` is not a good choice, duplicate.
2. In `copy_file_to`, it will get the available address again, if this ip address is not available for some reason, the following `wait_for_login` can not fix it too.

Reply:
The purpose to add wait_for_login() is to make sure vm starts up completely and ssh service work. I moved this to copy_network_script() before copy file. Thanks for review.